services/src/datasets/external/edr.rs

```diff
@@ -1705,13 +1705,27 @@ mod tests {
             .times(0..2)
             .respond_with(status_code(404)),
         );
-        let meta = load_metadata::<
-            GdalLoadingInfo,
-            RasterResultDescriptor,
-            RasterQueryRectangle,
-            PostgresDb<NoTls>,
-        >(&mut server, "GFS_isobaric!temperature!1000", ctx.db())
-        .await;
+
+        // TODO: This test is flaky in the CI, so we run it multiple times to increase the chance of success
```

Comment:
can you elobarate on the flakyness?

Reply:
[718c4e9](https://github.com/geo-engine/geoengine/pull/1030/commits/718c4e9bd5bb143329930c95413241158fcebdff)